.github/workflows/main-docker.yml

```diff
@@ -242,6 +242,17 @@ jobs:
           docker buildx imagetools create $(jq -cr '.tags | map("-t " + .) | join(" ")' <<< "$DOCKER_METADATA_OUTPUT_JSON") \
             -t ${{ env.DOCKERHUB_REGISTRY }}/${{ env.IMAGE_NAME }}:${REF_NAME} \
             $(printf '${{ env.DOCKERHUB_REGISTRY }}/${{ env.IMAGE_NAME }}@sha256:%s ' *)
+
+          # If the ref is a tag, also tag the image as stable as this is part of a 'release'
+          if [[ "${GITHUB_REF}" == refs/tags/* ]]; then
+            docker buildx imagetools create $(jq -cr '.tags | map("-t " + .) | join(" ")' <<< "$DOCKER_METADATA_OUTPUT_JSON") \
+              -t ${{ env.GHCR_REGISTRY }}/${{ env.IMAGE_NAME }}:stable \
+              $(printf '${{ env.GHCR_REGISTRY }}/${{ env.IMAGE_NAME }}@sha256:%s ' *)
+            
+            docker buildx imagetools create $(jq -cr '.tags | map("-t " + .) | join(" ")' <<< "$DOCKER_METADATA_OUTPUT_JSON") \
+              -t ${{ env.DOCKERHUB_REGISTRY }}/${{ env.IMAGE_NAME }}:stable \
+              $(printf '${{ env.DOCKERHUB_REGISTRY }}/${{ env.IMAGE_NAME }}@sha256:%s ' *)
+          fi
```

Comment:
You did quite a bit of bash magic in here. 😎 

Does this also handle the case where I push a beta version such as `v0.90.7-beta`? In this case we shouldn't tag it as `stable`; the `stable` tag should remain to `v0.90.4` until say `v0.90.8` comes along.

Reply:
Yeah it's not my favorite - and no, that's a good point. I was just catching if there was a new tag pushed. I'm assuming I would just have to catch if there is a `-` in the string of the tag that is pushed, to assume that it's not a `stable` build?